tests/logs.go

```diff
@@ -101,7 +101,7 @@ func Logs(o *option.Option) {
 					command.Run(o, "exec", testContainerName, "sh", "-c", fmt.Sprintf("echo %s >> /proc/1/fd/1", newLog))
 					// allow propagation time
 					gomega.Eventually(strings.TrimSpace(string(session.Out.Contents()))).
-						WithTimeout(15 * time.Second).
+						WithTimeout(30 * time.Second).
```

Comment:
seems it took [FAILED] [28.582 seconds] which is close to 30 seconds, can we add more buffer? like 45 sec or 1 min. Not sure this is a deterministic time and might involve how much resources are available at a given time.

Reply:
The failed test timestamp isn't a good indicator since it also does some cleanup. I think looking at successful tests is more representative of how long it actually might take